src/UI/Art/SpriteArtist/HorizontalSelectorTemplate.java

```diff
@@ -98,16 +99,26 @@ private BufferedImage generateOptionPanel(int length, Color fgColor)
 
   public void drawGameOption(Graphics g, int x, int y, GameOption<?> opt)
   {
-    int drawBuffer = textBuffer;
+    PixelFont pf = SpriteLibrary.getFontStandard();
+    int drawBuffer = pf.getDescent(); // Use the same buffer above and below the main text mass.
 
     // Draw the name panel and the name.
     g.drawImage(optionNamePanel, x, y, optionNamePanel.getWidth(), optionNamePanel.getHeight(), null);
-    SpriteUIUtils.drawText(g, opt.optionName, x + drawBuffer, y + drawBuffer);
 
+    // These debug boxes are being left here commented-out for future use by popular request.
+//    g.setColor(Color.GREEN);
+//    g.drawRect(x+drawBuffer, y+drawBuffer, pf.emSizePx, pf.getHeight());
+//    g.setColor(Color.RED);
+//    g.drawRect(x+drawBuffer, y+drawBuffer, pf.getWidth('H'), pf.getAscent());
+//    g.setColor(Color.YELLOW);
+//    g.drawRect(x+drawBuffer, y+drawBuffer+pf.getAscent(), pf.getWidth('H')*2, pf.getDescent());
+//    g.setColor(Color.BLACK);
+
+    SpriteUIUtils.drawText(g, opt.optionName, x + drawBuffer, y+drawBuffer);
     // Draw the setting panel and the setting value.
-    x = x + (optionNamePanel.getWidth() + optionSpacingPx);
+    x = x + (optionNamePanel.getWidth() + (3 * pf.emSizePx));
```

Comment:
I realize now that my `fontsMod` changes just made this line do a round trip back to its previous definition (but with a less useful variable name).
Why'd you delete `optionSpacingPx`? I added `horizontalSpacing` because it was useful for my arrow-positioning logic.

Reply:
I think because `optionSpacingPx` was defined relative to `letterWidth`, which I was replacing with Em size. I definitely tweaked the code here multiple times before getting something that worked. I'm definitely fine with putting a proper variable name here again and moving the magic 3 back to where it makes more sense.

The real question is probably: Why does every place that uses `HorizontalSelectorTemplate` have slightly different logic for iterating and drawing all of the options?

Reply:
That's a great question, and why I was proposing moving more of that logic into `HorizontalSelectorTemplate`.
I don't recall at the moment why we decided that wasn't a great idea, but maybe I should throw some annotation in on my new spacing logic in `fontsMod`?

Reply:
Sure